Python/Tests/Core/EnvironmentListTests.cs

```diff
@@ -771,6 +772,121 @@ public class EnvironmentListTests {
             TestFileNameEllipsis("", "", "", "");
         }
 
+        [TestMethod, Priority(0)]
+        public void FilterPython2Interpreter() {
+            PythonVersion pythonInterpreter =   PythonPaths.Python27_x64 ??
+                                                PythonPaths.Python26_x64 ??
+                                                PythonPaths.Python27 ??
+                                                PythonPaths.Python26;
+
+            Assert.IsNotNull(pythonInterpreter, "Python 2.6 or 2.7 not found");
+            FilterPythonInterpreterEnv(pythonInterpreter);
+        }
+
+        [TestMethod, Priority(0)]
+        public void FilterPython3Interpreter() {
+            PythonVersion pythonInterpreter =   PythonPaths.Python37_x64 ??
+                                                PythonPaths.Python36_x64 ??
+                                                PythonPaths.Python37 ??
+                                                PythonPaths.Python36;
+
+            Assert.IsNotNull(pythonInterpreter, "Python 3.6 or 3.7 not found");
+            FilterPythonInterpreterEnv(pythonInterpreter);
+        }
+
+        [TestMethod, Priority(0)]
+        public void FilterConda2Interpreter() {
+            PythonVersion condaInterpreter = PythonPaths.Anaconda27_x64 ?? PythonPaths.Anaconda27;
+
+            Assert.IsNotNull(condaInterpreter, "Anaconda 2.6 or 2.7 not found");
```

Comment:
In all these tests, if the machine doesn't have the required Python installed, we don't want to fail.

Use Assert.Inconclusive(), because we can't tell whether it would fail or not if it would be present.

This is what we do in all other tests.

Reply:
Actually, we have an extension method for that. Just call `condaInterpreter.AssertInstalled()`